momentum-design/momentum-design
# feat: added icons & illustrations build

# Description

- Adjusted the existing icons builder to be more abstract & named it assets builder
- Added file handler to assets builder
- Added svgo transform to the assets builder
- Added the assets-builder + config to the illustrations package (cause those are also svgs, which need to be transformed)
- Small adjustments to general files, like README.md, etc.

To test it locally:
1. Run `yarn` on root
2. Run `yarn builder build` on root
3. Run `yarn icons build` on root and then check the `dist` folder in icons package
4. Run `yarn illustrations build` on root and then check the `dist` folder in illustrations package

TODOs: 
- Add integration tests

## Links

https://github.com/svg/svgo
